client/src/router.tsx

```diff
@@ -24,6 +24,8 @@ import * as QueueRoute from './routes/queue';
 import * as RootRoute from './routes/root';
 import * as SettingsRoute from './routes/settings';
 import * as ShowRoute from './routes/shows/show';
+import * as SearchQueryRoute from './routes/search/query';
```

Comment:
Looks like this component is unused. Do you need this one anymore with the way you've structured the search page?

Reply:
See comment about the refactor in https://github.com/apollographql/spotify-showcase/commit/d2bd8bc6bdcd9d0ad6dd717e35d038bee1967a59, this page is used here now 👍